src/components/Disturber.tsx

```diff
@@ -78,14 +79,18 @@ export const Disturber = ({ navigation, publicJsonFile }: Props) => {
 
   if (!isVisible || !closestItem) return null;
 
+  const { showButtonToClose = true } = closestItem;
```

Comment:
with having destruction of `closestItem` here i suggest to add all of its used variables, like `backgroundColor` and so on

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/b9c4ceae1a0b0270e3498e20e1a157cb3b8d0d7c